lib/src/new-select/index.d.ts

```diff
@@ -0,0 +1,40 @@
+type SVG = string | (HTMLElement & SVGElement);
+type Space = "xxsmall" | "xsmall" | "small" | "medium" | "large" | "xlarge" | "xxlarge";
+type Margin = {
+  top?: Space;
+  bottom?: Space;
+  left?: Space;
+  right?: Space;
+};
+
+type OptionGroup = {
+  label: string;
+  options: Option[];
+};
+type Option = {
+  icon?: string | SVG;
+  label: string;
+  value: string;
+};
+
+type Props = {
+  label: string;
+  name: string;
+  value: string | string[];
+  options: Option[] | OptionGroup[];
+  helperText: string;
+  placeholder: string;
+  disabled: boolean;
+  optional: boolean;
+  searchable: boolean;
+  multiple: boolean;
+  onChange: (value: string | string[]) => void;
+  onBlur: (val: { value: string | string[]; error: string }) => void;
+  error: string;
+  margin: string | Margin;
+  size: "small" | "medium" | "large" | "fillParent";
```

Comment:
We can add a Size type like this:`type Size = "small" | "medium" | "large" | "fillParent";` and use if for the size property, like in the rest of components.
Comments are left but I suppose it is an initial version.

Reply:
This change respect to the other components has a purpose. If we use an in-between type, then when you press `Ctrl` key over the prop while consuming the component, you see the type Size but not its values, as this:

<img width="313" alt="Sin título 2" src="https://user-images.githubusercontent.com/44321109/144385375-afd5da04-38a8-4359-a522-f090559e7987.png">

While if you do it like above, you see them:

<img width="384" alt="Sin título" src="https://user-images.githubusercontent.com/44321109/144385571-5adbb4e9-592a-46ca-9cac-2b398afd0cf6.png">

I find this way more useful for the user, what do you think?